src/olympia/activity/tests/test_utils.py

```diff
@@ -225,25 +225,33 @@ def test_exception_but_spammy(self):
 class TestAddEmailToActivityLog(TestCase):
     def setUp(self):
         self.addon = addon_factory(name='Badger', status=amo.STATUS_NOMINATED)
-        version = self.addon.find_latest_version(channel=amo.CHANNEL_LISTED)
+        self.version = self.addon.find_latest_version(channel=amo.CHANNEL_LISTED)
         self.profile = user_factory()
-        self.token = ActivityLogToken.objects.create(version=version, user=self.profile)
+        self.token = ActivityLogToken.objects.create(
+            version=self.version, user=self.profile
+        )
         self.token.update(uuid='5a0b8a83d501412589cc5d562334b46b')
         self.parser = ActivityEmailParser(sample_message_content['Message'])
+        user_factory(id=settings.TASK_USER_ID)
+        assert not self.version.due_date
 
     def test_developer_comment(self):
         self.profile.addonuser_set.create(addon=self.addon)
         note = add_email_to_activity_log(self.parser)
         assert note.log == amo.LOG.DEVELOPER_REPLY_VERSION
         self.token.refresh_from_db()
         assert self.token.use_count == 1
+        assert self.version.needshumanreview_set.exists()
+        assert self.version.reload().due_date
```

Comment:
Could test it's the right one too since the delay is custom (maybe including another test where it already had one, making sure we aren't overriding it)

Reply:
we have those tests for the viewset, but we could duplicate them here too